qi-lib/flow/core/deforest.rkt

```diff
@@ -260,6 +260,25 @@
                    ((#%host-expression (~datum list-ref)) __ idx)))
              #:attr end #'(cad*r-cstream-next idx))
 
+    (pattern (~or (esc
```

Comment:
Re: commit message for 1f12a2b540b057edfed6d6204fb563461de62276 (and others, I think): Prefer imperative subjects (see `$(git --html-path)/SubmittingPatches.html` and
https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).